src/server/index.jsx

```diff
@@ -47,18 +48,43 @@ const publicDirectory = getPublicDirectory();
 const dataFolderToRender =
   process.env.NODE_ENV === 'production' ? 'data/prod' : 'data/test';
 
+const articleDataRegexPath = `${articleRegexPath}.json`;
+
 const server = express();
 server
   .disable('x-powered-by')
   .use(compression())
   .use(helmet.frameguard({ action: 'deny' }))
-  .use('/data', express.static(dataFolderToRender))
   .use(
     expressStaticGzip(publicDirectory, {
       enableBrotli: true,
       orderPreference: ['br'],
     }),
   )
+  .get(articleDataRegexPath, async ({ params }, res) => {
+    const { service, id } = params;
+
+    const dataFilePath = path.join(
+      dataFolderToRender,
+      service,
+      'articles',
+      `${id}.json`,
+    );
+
+    fs.readFile(dataFilePath, (error, data) => {
+      if (error) {
+        res.sendStatus(404);
+        console.log(error); // eslint-disable-line no-console
+        return null;
+      }
+
+      const articleJSON = JSON.parse(data);
+
+      res.setHeader('Content-Type', 'application/json');
+      res.json(articleJSON);
+      return null;
```

Comment:
Could you elaborate on why we need to return null here?

Reply:
The callback needs to return something else linting complains, but we don't care what the value is because we should have sent a response.